### ISSUE TITLE
Add `response_mime_type` support to `generationConfig`

### ISSUE THREAD

**Issue by DrAliRagab** [gemini-api-php/client#31]
According to google docs here https://ai.google.dev/docs/gemini_api_overview#json
We can instruct Gemini to return `json` by adding `"response_mime_type": "application/json"` to `generationConfig`

I hope this will be added to `generationConfig`

**Comment by Martin-Matchory** [comment]
+1
... and the corresponding `responseSchema` please
example:

`"responseMimeType" => "application/json",
"responseSchema" => [
                    "type" => "ARRAY",
                    "items" => [
                        "type" => "OBJECT",
                        "properties" => [
                            "product" => [
                                "type" => "STRING"
                            ]
                        ]
                    ]
                ]`

**Comment by benbowler** [comment]
I noticed [this library has an open tested PR for mimetype support](https://github.com/google-gemini-php/client/pull/23/files) so I'll close this one.

**Comment by DrAliRagab** [comment]
> +1 ... and the corresponding `responseSchema` please example:
> 
> `"responseMimeType" => "application/json", "responseSchema" => [ "type" => "ARRAY", "items" => [ "type" => "OBJECT", "properties" => [ "product" => [ "type" => "STRING" ] ] ] ]`

Yes, both `responseMimeType` and `responseSchema` should be added.

This is a working example using laravel http:
```php
$response = Http::baseUrl($this->baseUrl)
->withHeaders([
    'x-goog-api-key' => $this->apiKey,
])
->asJson()
->acceptJson()
->post("v1beta/models/{$this->model}:generateContent", [
    'contents'         => [
        'parts' => [
            [
                'text' => $instructions,
            ],
        ]
    ],
    'generationConfig' => [
        'responseMimeType' => 'application/json',
        'responseSchema'   => [
            'type'       => 'object',
            'properties' => [
                'score' => [
                    'type'    => 'number',
                    'minimum' => 0,
                    'maximum' => 1,
                    'format'  => 'float',
                ],
            ],
            'required'   => ['score'],
        ],
    ],
]);

```

**Comment by evandromurilo** [comment]
#65
